bolt.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = "~> 2.0"
 
-  spec.add_dependency "addressable", "~> 2.4"
+  spec.add_dependency "addressable", '< 2.5.0'
```

Comment:
I think it might be better to pin back `public_suffix` as `< 3`.

Addressable releases are:

* 2.4.0 - 12/7/2015
* 2.5.0 - 10/4/2016 - added Ruby 2.4 support, UTF-8 encoding fixes, and a number of other additions
* 2.5.1 - 3/29/2017 - configurable normalization
* 2.5.2 - 8/25/2017 - bug fixes

Full changelog at https://github.com/sporkmonger/addressable/blob/master/CHANGELOG.md